src/validator/mod.rs

```diff
@@ -182,6 +182,8 @@ impl Validator {
         Ok(sequence_number)
     }
 
+    /// Returns `Some` with the Ctip for the given sidechain number. `None`
+    /// if there's no Ctip for the given sidechain number.
```

Comment:
This should be renamed to `try_get_ctip` to make it clear that a ctip may not exist